docs/embedding/full-app-embedding.md

```diff
@@ -84,17 +84,23 @@ You must URL encode (or double encode, depending on your web setup) all of the p
 https://metabase.example.com/auth/sso?jwt=<token>&redirect=%2Fdashboard%2F1%3Ffilter1%3Dvalue%26filter2%3Dvalue
 ```
 
+## Cross-browser compatibility
+
+To make sure that your embedded Metabase works in all browsers (including iOS and Safari), put Metabase and the embedding app in the same top-level domain (TLD).
```

Comment:
Can you elaborate on this? Just another clause or sentences on what you mean by top-level domain.